### PR TITLE
LPS-111104 Log4j test

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang The PR finished portal used log4j function.
These function included the below use cases.
1. Log4JUtil invoked log4j function. (read configure, setLevel, getCurrentLoggers)
2. Log4jLogImpl invoked log4j function. (logger.isEnabled, log messages).
3. Log4j's RollingFileAppender and ConsoleAppender.

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
Fetched

File: modules/apps/portal/portal-log4j-test/src/test/java/com/liferay/portal/log4j/Log4jTest.java [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+			"logMessage should be " + expectedOutput,
+			expectedOutput.equals(sb.toString()));
+	}
+
+	private static final String[] _NAMES = {
+		"level", "level.off", "level.fatal", "level.error", "level.warn",
+		"level.info", "level.debug", "level.trace"
+	};
+
+	private static final Log _log = LogFactoryUtil.getLog(Log4jTest.class);
```
[the remark this answers is not in the document]
Comment (dantewang) [reply modules/apps/portal/portal-log4j-test/src/test/java/com/liferay/portal/log4j/Log4jTest.java:460]: OK at first it's a JDK logger. I didn't look into your test in depth, could you clarify what is this logger for? Will it be replaced to a Log4J logger after your setup?

**Pull request closed without merging**